recipe/meta.yaml

```diff
@@ -43,6 +43,7 @@ requirements:
     - pyqt  # [not osx]
     - tk 8.5*  # [linux]
     - functools32  # [py2k]
+    - subprocess32  # [py2k and unix]
```

Comment:
Did I mark the selectors up correctly?  I started to take a run at packaging this last night, but gave up and went to sleep.